src/main/java/uk/gov/hmcts/reform/sscs/json/SscsJsonExtractor.java

```diff
@@ -35,7 +35,9 @@ public ScannedData extractJson(Map<String, Object> exceptionCaseData) {
 
             JSONObject jsonObject = (JSONObject) j.get(VALUE);
 
-            pairs.put(jsonObject.get(KEY).toString(), jsonObject.get(VALUE).toString());
+            String jsonValue = jsonObject.has(VALUE) ? jsonObject.get(VALUE).toString() : null;
```

Comment:
can jsonObject.has(VALUE) return null it might result in NPE ?

Reply:
There is a test in SscsJsonExtractorTest covering this exact scenario and no NPE is thrown so think we are fine